workshops/spinning_wheel/README.md

```diff
@@ -0,0 +1,434 @@
+
+Create another div with `id="box"` `class="box"`. Nest this inside the mainbox div by putting it in-between `<div ...>` and `</div>`.
+Nesting is like Russian [Matryoshka dolls](https://en.wikipedia.org/wiki/Matryoshka_doll), so take care of it.
+
+Now nest two more divisions in `<div id="box" class="box">` with class as “box1” and “box2” respectively. Just place both of these in between that `<div ...>` and `</div>`.
+Now in the `<div class="box1">` tag, use 4 span tags one by one and give the class as “span1”, “span2”, “span3”, and “span4” respectively.
+
+```html
+<div class="box1">
+  <span class="span1"></span>
+  <span class="span2"></span>
+  <span class="span3"></span>
+  <span class="span4"></span>
+</div>
+```    
+
+Right now, your division should look something like this.
+
+Then inside those span tags, you can write your option and put them inside (`<b> </b>`) tags to bold your font.
+
+Now we will copy all the 4 spans and will paste it in the next division having `<div class="box2">` tag. Then just change the options of your wheel for this division.
+
+We have created all the options, now we will just add the `spin` button which we saw in the center. For that just write (`<button class="spin" onclick="myfunction()">SPIN</button>`) before the ending of your last division tag. Here we have used the onclick function that will be explained in the JavaScript part of our workshop.  
+```html
+<html>
+  <head>
+    <title>Movie Selector</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body> 
+  <div id="mainbox" class="mainbox">
+    <div id="box" class="box">
+      <div class="box1">
+        <span class="span1"><b>Iron Man</b></span>
+        <span class="span2"><b>7500</b></span>
+        <span class="span3"><b>Bat Man</b></span>
+        <span class="span4"><b>Joker</b></span>
+      </div>
+      <div class="box2">
+        <span class="span1"><b>Shoplifters</b></span>
+        <span class="span2"><b>Inception</b></span>
+        <span class="span3"><b>Deadpool</b></span>
+        <span class="span4"><b>Terminator</b></span>
+      </div>
+    </div> 
+    <button class="spin" onclick="myfunction()">SPIN</button>
+  </div>
+  <script src="script.js"></script>
+  </body>
+</html>
+```
+
+Your final code in the index.html should look something like this. Woot woot! We are all done with the HTML part of our project.
+
+## 2)	CSS
+After you have done the writing part of HTML, you'll notice that when you click the "run" button at the top, you just see a white screen with some text("options") with Spin Button. Let's now add some styles to our elements and make them look better like the way we want it to be.🤩
+ 
+Firstly, let’s go to style.css.
```

Comment:
"Firstly" is not a word. "First" I think is what should be here. I think this sentence should say "First, navigate to the `style.css` file on your sidebar."